docs/tutorial.md

```diff
@@ -482,6 +492,9 @@ PaSh’s library currently several aggregators, many of which are usable by more
 
 ## Experimental Evaluation
 
+_Most benchmark sets in the evaluation infrastructure include a `input/setup.sh` script for fetching inputs and setting up the experiment appropriately._
+See [Running other script]() later.
+
 This section provides detailed instructions on how to replicate parts of the experimental evaluation of the system.
 
 Note that input files that are used as inputs for this script are generated using the `gen*` scripts in [evaluation/scripts/input/](../evaluation/scripts/input/).
```

Comment:
This is not correct anymore right?

Reply:
Yeah! I haven't started a pass over the evaluation yet though, this is just what was there already.

Reply:
oh cool nice